pkgs/development/compilers/mlton/default.nix

```diff
@@ -83,6 +85,11 @@ stdenv.mkDerivation rec {
     for e in mllex mlyacc ; do
       patchelf --set-interpreter ${dynamic_linker} $(pwd)/../${usr_prefix}/bin/$e
     done
+  '' + stdenv.lib.optionalString stdenv.isDarwin ''
+    # Patch libgmp linking
```

Comment:
I don't see any references to `/opt/local` I assume this is because of the macports source